src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java

```diff
@@ -90,10 +88,8 @@ public SBApiResponse registerUser(UserRegistrationInfo userRegInfo) {
 		String errMsg = validateRegisterationPayload(userRegInfo);
 		if (StringUtils.isBlank(errMsg)) {
 			try {
-				if (isUserExist(Constants.EMAIL, userRegInfo.getEmail().toLowerCase())) {
+				if (userUtilityService.isUserExist(userRegInfo.getEmail().toLowerCase())) {
 					errMsg = Constants.EMAIL_EXIST_ERROR;
-				} if (isUserExist(Constants.PHONE, userRegInfo.getPhone())) {
-					errMsg = Constants.PHONE_NUMBER_EXIST_ERROR;
```

Comment:
why this condition is been removed -- this is going to affect direct user registration flow